testgrid/config/config.yaml

```diff
@@ -2674,78 +2581,35 @@ dashboards:
   - name: periodic-bazel-test-1.7
     test_group_name: periodic-kubernetes-bazel-test-1-7
 
+# These are the release *blocking* tests.  These provide a valid binary signal
+# to gate releases (alpha, beta, official).
+# This list is used by kubernetes/release/lib/releaselib.sh to provide no/nogo
+# signal for releases.
+# The first test/suite in the list is the primary suite for release purposes
 - name: release-1.7-blocking
   dashboard_tab:
-  - name: build-1.7
```

Comment:
Let's keep the build test? Actually i think the build test should be the primary test. cc @krzyzacy